tsl/test/expected/compress_auto_sparse_index.out

```diff
@@ -26,9 +26,9 @@ select count(compress_chunk(x)) from show_chunks('sparse') x;
 explain select * from sparse where value = 1;
                                                        QUERY PLAN                                                        
 -------------------------------------------------------------------------------------------------------------------------
- Custom Scan (DecompressChunk) on _hyper_1_1_chunk  (cost=0.27..19.75 rows=72000 width=12)
+ Custom Scan (DecompressChunk) on _hyper_1_1_chunk  (cost=0.27..20.20 rows=76000 width=12)
```

Comment:
Not really sure what happened here. How could we get 4 more rows?

Reply:
Oh that's a bad test, probably I wanted to write `explain (analyze, costs off, timing off)` or at least `costs off`. These are the cost estimates, not the real number of rows.